tcms/testruns/tests/test_views.py

```diff
@@ -561,24 +561,6 @@ def test_404_if_edit_non_existing_run(self):
 
         self.assert404(response)
 
-    def test_edit_run(self):
```

Comment:
The only test method that is left is the `test_404_if_edit_non_existing_run()` which is not needed anymore. After the refactoring to generic CBV the 404 is handled internally by the parent class and we don't bother with it. So there's no need to actually test how Django internals work. Delete the entire test class.


Note: in views where we try-except the situation of non-existing objects and return the 404 ourselves such tests are still needed and relevant.